compiler/modules/front/codeGen.bal

```diff
@@ -1338,9 +1333,8 @@ function codeGenExpr(CodeGenContext cx, bir:BasicBlock bb, Environment env, s:Ex
             return codeGenFieldAccessExpr(cx, bb, env, pos, container, fieldName);
         }
         // List construct
-        // JBUG #33309 should be able to use just `var { members }`
-        var listConstructorExpr if listConstructorExpr is s:ListConstructorExpr => {
-            return codeGenListConstructor(cx, bb, env, listConstructorExpr);  
+        var { startPos, endPos, members, opPos, expectedType } => {
+            return codeGenListConstructor(cx, bb, env, startPos, endPos, members, opPos, expectedType);
```

Comment:
I don't think this is an improvement, because we can't then use s:range n codeGenListConstructor. Just delete the JBUG.

I anticipate changing this function to use if/else rather than match.

Reply:
fixed with https://github.com/ballerina-platform/nballerina/pull/847/commits/73e0700ab8c7d6c0c88a8b2cd34af18af27e6a1c